utility/BUILD.bazel

```diff
@@ -1,42 +1,40 @@
-# TODO: Rename file to 'BUILD' post-Gradle. See https://github.com/oppia/oppia-android/issues/1532.
+# TODO(#1532): Rename file to 'BUILD' post-Gradle.
+'''
+This library contains utilities that all other modules, minus model, depend on.
+It also contains Robolectric and JUnit tests for it's utilities.
+In Bazel, Kotlin source files are built into a library using the kt_android_library() rule.
+Kotlin test files must be built in their own kt_android_library() rule and added as a dependency to
+an android_local_test() rule which configures instrumentation tests in Bazel.
+'''
+
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
 load("@rules_jvm_external//:defs.bzl", "artifact")
 load("@dagger//:workspace_defs.bzl", "dagger_rules")
 
-'''A kt_android_library() rule that builds utility module kotlin source files into a library.'''
+# A kt_android_library() rule that builds utility module kotlin source files into a library.
 kt_android_library(
-    name = "utility_lib",
+    name = "utility",
     custom_package = "org.oppia.util",
-    srcs = glob(["src/main/java/org/oppia/util/**/*.kt"],
-                exclude = ["src/main/java/org/oppia/util/logging/firebase/*.kt"]),
+    srcs = glob(["src/main/java/org/oppia/util/**/*.kt"]),
     resource_files = glob(["src/main/res/**/*.xml"]),
     manifest = "src/main/AndroidManifest.xml",
     deps = [
         ":dagger",
-        "//model:model_lib",
+        "//model:model",
```

Comment:
Does just ‘//model’ work? Generally the extra : qualification isn’t needed when it matches the directory name, but it might be for top level targets.

Reply:
Yes //model works! I will go ahead and change that wherever its applicable.